cli.go

```diff
@@ -55,19 +62,25 @@ func PlugCLI() (string, parser.RenpyGraphOptions) {
 	flag.BoolVar(&hideScreens, "hide-screens", hideScreens, "Hide screens")
 	flag.BoolVar(&hideNestedScreens, "hide-nested", hideNestedScreens, "Hide nested screens")
 	flag.BoolVar(&fullDebug, "debug", fullDebug, "Debug")
+	flag.StringVar(&skipFilesRegex, "skip-files", skipFilesRegex, "Regex pattern for excluding files")
 
-	flag.Parse()
-
-	path := "."
-	if len(flag.Args()) > 0 {
-		path = flag.Args()[0]
+	// Manually handle the non-flag argument
+	var path string
+	if len(os.Args) > 1 && !isFlag(os.Args[1]) {
+		path = os.Args[1]
+		os.Args = append(os.Args[:1], os.Args[2:]...) // Remove the non-flag argument from os.Args
+	} else {
+		path = "."
 	}
+	flag.Parse()
```

Comment:
`flag.Parse()` does it too, why are you doing it manually?

Reply:
Because if it already did that, the bug would not be happening in a first place, so because the bug occurs with just calling flag.Parse and does not occur after this path extraction, it means the flag.Parse does not do it on it's own. It actually skips the arg, because it recognizes only named arguments and skips the rest.